Update README.md (#39)
iotaledger/chrysalis-docs: docs/guides/README.md

* The concepts of `bundles` and `transactions` were replaced with the concepts of `messages` and `payloads`. The `message` is a data structure that is actually being broadcast in the network and represents a node (vertex) in the Tangle graph. See [messages, payload and transactions](./dev_guide.md#messages-payload-and-transactions) and [selected message payloads](./dev_guide.md#selected-message-payloads)
* The IOTA network is based on a DAG (Directed Acyclic Graph) to store individual `messages` (and related `transactions`). However, each `message` can newly reference up to 8 parent messages. See [messages, payload and transactions](./dev_guide.md#messages-payload-and-transactions)
* The signature scheme based on `WOTS` was replaced with with `Ed25519` signature scheme. See [seed and addresses](./dev_guide.md#seed-and-addresses)

> * Due to the changed signature scheme, IOTA addresses are reusable without any negative security impact
> * In contrary to IOTA 1.0 that was based on ternary, IOTA 1.5 is based on binary and, thus, very efficient on all kinds of current hardware devices

Reviewer: ```suggestion
* Due to the changed signature scheme, IOTA addresses are reusable without any negative security impact
* In comparison to IOTA 1.0, which was based on ternary, IOTA 1.5 is based on binary and is thus very efficient on all kinds of current hardware devices
```
